packages/sit-onyx/src/composables/useCustomValidity.ts

```diff
@@ -5,11 +5,13 @@ import enUS from "../i18n/locales/en-US.json";
 import { injectI18n } from "../i18n";
 import type { InputType } from "../components/OnyxInput/types";
 
+export type CustomErrorType = string | FormErrorMessages;
+
 export type CustomValidityProp = {
   /**
    * Custom error message to show. Will only show up after the user has interacted with the input.
    */
-  customError?: string;
+  customError?: CustomErrorType;
```

Comment:
Can be update the Storybook controls to be of type text so the user can play around with it?
![image](https://github.com/SchwarzIT/onyx/assets/67898185/9dedd3db-6d5f-4831-8730-4dd0b4c370d6)

Reply:
if I set text, storybook can't interpret the example
![image](https://github.com/SchwarzIT/onyx/assets/151019977/c3cbc7d5-a550-45d3-ad5e-206f3d46862d)

if I leave the standard, it works for object + string aswell
![image](https://github.com/SchwarzIT/onyx/assets/151019977/d0ffda33-6963-4b17-9247-b9b8c6a5c0f5)
![image](https://github.com/SchwarzIT/onyx/assets/151019977/3c8b17a4-da27-4dec-873c-c40f173abf1f)

when you edit raw, you can just put in a string